### PR TITLE
DMP-2133: Added urgeny json block when getting transcriptions information

### PR DESCRIPTION
**Pull request by mestebanez**
### JIRA link (if applicable) ###

[DMP-2133](https://tools.hmcts.net/jira/browse/DMP-2133)

### Change description ###

Added an urgency json block when using the restful transcription api to get transcriptions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No -Backward compatibility maintained as mentioned in the ticket
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/darts/transcriptions/enums/TranscriptionUrgencyEnum.java [hunk 1]
```diff
@@ -9,15 +9,17 @@
...
 @Getter
 public enum TranscriptionUrgencyEnum {
-    STANDARD(1),
-    OVERNIGHT(2),
-    OTHER(3),
-    WORKING_DAYS_3(4),
-    WORKING_DAYS_7(5),
-    WORKING_DAYS_12(6),
-    WORKING_DAYS_2(7);
+    STANDARD(1, 999, "Standard"),
```
Comment (cmagowan-hmcts) [inline src/main/java/uk/gov/hmcts/darts/transcriptions/enums/TranscriptionUrgencyEnum.java:12]: we seem to be replicating the data in the database here. i'm not sure thats a good plan. would that make the data in the database redundant?

Comment (davet1985) [reply src/main/java/uk/gov/hmcts/darts/transcriptions/enums/TranscriptionUrgencyEnum.java:12]: Yeah good question @cmagowan-hmcts. @mestebanez any specific reason behind this change?

Comment (mestebanez) [reply src/main/java/uk/gov/hmcts/darts/transcriptions/enums/TranscriptionUrgencyEnum.java:12]: yep assumed this enum was specifically for test purposes. I am changing to read urgency details from the database

**Review by davet1985 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapper.java [hunk 2]
```diff
@@ -152,7 +153,15 @@ public GetTranscriptionByIdResponse mapToTranscriptionResponse(TranscriptionEnti
         }
         if (nonNull(transcriptionEntity.getTranscriptionUrgency())) {
             transcriptionResponse.setUrgency(transcriptionEntity.getTranscriptionUrgency().getDescription());
+
+            // populate the urgency details
+            TranscriptionUrgencyDetails urgencyDetails = new TranscriptionUrgencyDetails();
+            urgencyDetails.setPriorityOrder(transcriptionEntity.getTranscriptionUrgency().getPriorityOrder());
+            urgencyDetails.setTranscriptionUrgencyId(transcriptionEntity.getTranscriptionUrgency().getId());
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapper.java:160]: The setUrgency line above (line 155) that this new field is superseding has a null check on transcription urgency, do we need a similar check here?

**Pull request merged**